tox.ini

```diff
@@ -13,10 +13,8 @@ PLATFORM =
 
 [testenv]
 deps =
+    -rrequirements.txt
```

Comment:
Actually neither `djangorestframework` nor `django-tastypie` should be required for this project. Maybe we should move both of those packages to `requirements_test.txt` ?

Reply:
Yeah, that is true